node_modules/col-assets/tests/test-assets.js

```diff
@@ -79,6 +80,68 @@ describe('Assets', function() {
       });
     });
 
+    describe('File', function() {
+
+      /**
+       * Get a file stream
+       * @param  {String}   filename    The name of the file in the `data` directory
+       * @return {Stream}               A readable stream to the file on disk
+       */
+      var getFileStream = function(filename) {
+        return fs.createReadStream(__dirname + '/data/' + filename);
+      };
+
+      /**
+       * Test that verifies that a new file asset can be created
```

Comment:
In the `getAsset` and `getAssets` tests, we should also create a file to ensure that the expected properties are returned